templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="{% url 'taxi:index' %}"></a>Home page</li>

```

Comment:
The 'Home page' text should be inside the anchor tag for it to be clickable. It should be: `<a href="{% url 'taxi:index' %}">Home page</a>`.